test/person_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../person'
+
+describe Person do
+  let(:person_one) { Person.new(27, 'Isma') }
+  let(:person_two) { Person.new(4, 'Manel', parent_permission: false) }
+  let(:person_three) { Person.new(20, 'Sarah') }
+  let(:person_four) { Person.new(15, 'Sissa', parent_permission: false) }
+
+  describe '#can_use_services?' do
```

Comment:
- [ ] You only tested for a single method (can_use_services?) in the class Person. kindly add tests for other methods like `add_rental`, `correct_name`, and other scenarios like validate_name, creating a new instance of the Person class.  
- You can also refer to [this guide](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/rspec_unit_test.md#unit-testing) for unit testing best practices.